### PR TITLE
add 2 new repos and added renamed repos

### PR DESCRIPTION
**Pull request by CoinClubQuincy**
I added 2 additional repos to the xdc.toml file and fixed 2 that were renamed

### CODE REVIEW

**Review by eherrerosj — APPROVED** [review]
Let some comment

**Review by eherrerosj — CHANGES_REQUESTED** [review]
can you remove the "/activity" URL suffix from the end of the 2 new repo URLs please?

**Comment by CoinClubQuincy** [comment]
just fixed it thank you

**Comment by CoinClubQuincy** [comment]
I added additional repos as well

File: data/ecosystems/x/xdc.toml [hunk 1]
```diff
@@ -657,3 +656,78 @@ url = "https://github.com/XSwapProtocol/XSP"
...
+
+[[repo]]
+url = "https://github.com/tkeith/airtracker"
+
+[[repo]]
+url = "https://github.com/qi-protocol/proof-of-something"
+
+[[repo]]
+url = "https://github.com/XinFinOrg/XDC-Helpers
+"
```
Comment (eherrerosj) [inline data/ecosystems/x/xdc.toml:707]: please move this double quote to the previous line

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a comment, validator not passing due to this

**Review by eherrerosj — APPROVED** [review]
thanks!

**Comment by eherrerosj** [comment]
Hi @CoinClubQuincy, we wanted to give you a heads-up that we are planning to freeze the data set that will be used for the analysis during the next few days. If you want your changes to be included and have an impact on this year’s report, please ensure that the requested modifications are made and merged prior to the data freeze.

Let's aim to have your PR finalized as soon as possible to make it into this window. If you have any questions or need further clarification, feel free to reply here or reach out directly.

**Pull request merged**